### ISSUE TITLE
[HelpEditor] Not saving changes

### ISSUE THREAD

**Issue by christinerogers** [aces/Loris#6940]
It seems the Help Editor isn't working as it was intended -- test report by Alex: 

_Originally posted by @AlexandraLivadas in https://github.com/aces/Loris/pull/6907#issuecomment-674284554_

The Help Editor does not correctly save changes made in the frontend to the file contents. I made changes and hit `Save`. The changes displayed correctly but once I hit `Return to Help Editor` or navigated back to the main page myself, the changes were actually not saved.

Another issue is that the button `Return to Help Editor` does not actually return me to the main Help Editor page. Instead, I stayed on the editing content page and the changes I made were not saved.

**Comment by laemtl** [comment]
Console output: Uncaught SyntaxError: Cannot use import statement outside a module
Return to Help Editor does work fine if Save is not hit first.

**Comment by AlexandraLivadas** [comment]
@zaliqarosli I'm pretty sure this issue was resolved by #6907.

**Comment by zaliqarosli** [comment]
i dont remember the details of this, but it looks like it! @AlexandraLivadas

**Comment by zaliqarosli** [comment]
just looked into the details of this. this issue was indeed fixed by #6907 - the changes are saving, and the `Return to Help Editor` button is working (on main). There is still an issue with the swal being triggered on clicking the 'Save' button:

```
help_editor_helper.js:66 Uncaught TypeError: swal.fire is not a function
    at Object.success (help_editor_helper.js:66:18)
    at j (jquery-1.11.0.min.js:2:27136)
    at Object.fireWith [as resolveWith] (jquery-1.11.0.min.js:2:27949)
    at x (jquery-1.11.0.min.js:4:22244)
    at XMLHttpRequest.b (jquery-1.11.0.min.js:4:26298)
```

but this issue is being targeted by https://github.com/aces/Loris/pull/7018 which is yet to the merged.
so @driusan we can just wait for 7018 to be merged for this issue to close? or should i close it now given that https://github.com/aces/Loris/issues/6114 is also open as an issue that 7018 is targetting?

**Comment by zaliqarosli** [comment]
was not fixed by 7018
